packages/config/config/devices/0x014a/tstat-tbz500.json

```diff
@@ -0,0 +1,625 @@
+{
+	"manufacturer": "Ecolink",
+	"manufacturerId": "0x014a",
+	"label": "TBH500",
+	"description": "Z-Wave Smart Thermostat",
+	"devices": [
+		{
+			"productType": "0x5442",
+			"productId": "0x5439"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "System Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Standard",
+					"value": 0
+				},
+				{
+					"label": "Heat Pump",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Fan Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Gas (No fan w/Heat)",
+					"value": 0
+				},
+				{
+					"label": "Electric (Fan w/Heat)",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "Change Over Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "w/Cool",
+					"value": 0
+				},
+				{
+					"label": "w/Heat",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "4",
+			"label": "2nd Stage Heat Enable",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "5",
+			"label": "Aux Heat Enable",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"defaultValue": 1
+		},
+		{
+			"#": "6",
+			"label": "2nd Stage Cool Enable",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
```

Comment:
We avoid "Enable/Disable" in the labels. The available options (here Enable/Disable) should be enough.

Reply:
Got it. Will change. Similar issue with the one I used as a base: `gocontrol_gc-tbz48.json`